### PR TITLE
Creating a `CODE OF CONDUCT` for the project

### PR DESCRIPTION
**Pull request by xeho91**
**This Pull Request intention is to open a discussion and make a decision on our "Code of Conduct" for this project.**

---

## TODO

- [x] Create a [CODE_OF_CONDUCT.md] file
- [x] Mention this in the [CONTRIBUTING.md] file
- [x] Add a shield _(badge)_ with project Code of Conduct
- [x] Insert our contact email to reach project leaders for reporting breaches

[CODE_OF_CONDUCT.md]: ./CODE_OF_CONDUCT.md
[CONTRIBUTING.md]: ./CONTRIBUTING.md

### CODE REVIEW

**Comment by xeho91** [comment]
**My thoughts:**

I want to adopt the most reasonable Code of Conduct for the open-source project - [Contributor Covenant].

[Contributor Covenant]: https://www.contributor-covenant.org/

What we need to do, is to decide on our email contact, to insert it into the proposed Code of Conduct. We'll discuss this matter further on our Discord server.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail
+address, posting via an official social media account, or acting as an
+appointed representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (lax0248917) [inline CODE_OF_CONDUCT.md:63]: Do you think we should have a single point of contact?

**Review by lax0248917 — APPROVED** [review]
This is perfect, and as the community grows, we need to implement something very similar in the discord server.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (xeho91) [reply CODE_OF_CONDUCT.md:63]: Assuming the project leaders may change over time, would it make sense to update the file every time?

During the transition, we would provide access to this single point of contact.

Comment (lax0248917) [reply CODE_OF_CONDUCT.md:63]: We could do what we did before and create a team email address, and manage access as the team grows. Thoughts?

Comment (xeho91) [reply CODE_OF_CONDUCT.md]: Done, the email will be terminal-nerds@pm.me.

**Review by lax0248917 — APPROVED** [review]
LGTM! Was able to log into the primary POC email account

**Comment by xeho91** [comment]
This Pull Request needs #42 to be completed first.

**Pull request merged**